specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the RecoveryServicesBackup API.
 title: Recovery Services Backup Client
 description: Open API 2.0 Specs for Azure RecoveryServices Backup service
 openapi-type: arm
-tag: package-2022-06-01-preview
+tag: package-preview-2022-09
```

Comment:
Why not follow previous convention package-2022-06-01-preview?

Reply:
Hi @raych1 the tool that we used to generate the new version on top of previous followed this convention. Should I update it to the older format ?